pyomo/scripting/commands.py

```diff
@@ -123,7 +122,7 @@ def pyomo_python(args=None):
         console.interact('Pyomo Python Console\n'+sys.version)
     else:
         cmd = sys.executable+' '+ ' '.join(args)
-        pyutilib.subprocess.run(cmd, stdout=sys.stdout, stderr=sys.stderr)
+        subprocess.run(cmd, stdout=sys.stdout, stderr=sys.stderr)
```

Comment:
Why are we specifying `stdout=sys.stdout, stderr=sys.stderr`? Shouldn't we just omit those arguments?

Reply:
Good point - I forgot that pyutilib's default was to suppress and Python's default is to print out.